stablehlo/conversions/tosa/stablehlo_tosa_opt.cc

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2020 OpenXLA Authors. All Rights Reserved.
```

Comment:
Should we perhaps merge stablehlo-tosa-opt into stablehlo-opt? In my mind, StableHLO to TOSA would be a core part of the project, so I'd love to have the passes in the main binary (similarly to how I think we should run tests as part of our CI).

Reply:
That's a good idea, let me do that.